src/main/java/com/delphix/masking/initializer/pojo/SystemInformation.java

```diff
@@ -0,0 +1,30 @@
+package com.delphix.masking.initializer.pojo;
```

Comment:
I think that this can be autogenerated?

You should be able to do something like
```java
  - name: profileExpressionNames
    type: list
    listType: String
```
to generate the list.